vignettes/Introduction_To_BiocParallel.Rmd

```diff
@@ -0,0 +1,867 @@
+---
+title: "**Introduction to *BiocParallel***"
+author:
+- name: "Valerie Obenchain"
+- name: "Vincent Carey"
+- name: "Michael Lawrence"
+- name: "Martin Morgan"
+  email: Martin.Morgan@RoswellPark.org
+date: "Edited: January 23, 2021; Compiled: `r format(Sys.time(), '%B %d, %Y')`"
+date: "Compiled: 18 October, 2022"
+package: BiocParallel
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document
+  
+---
+
+Introduction
+============
+
+Numerous approaches are available for parallel computing in R. The CRAN Task View for
+high performance and parallel computing provides useful high-level summaries and [package
+categorization.](http://cran.r-project.org/web/views/HighPerformanceComputing.html)
+Most Task View packages cite or identify one or more of [*snow*](https://cran.r-project.org/package=snow) , [*Rmpi*](https://cran.r-project.org/package=Rmpi),[*multicore*](https://cran.r-project.org/package=multicore)  or [*foreach*](https://cran.r-project.org/package=foreach) as
+relevant parallelization infrastructure. Direct support in *R* for *parallel* computing started
+with release 2.14.0 with inclusion of the [parallel](https://cran.r-project.org/package=parallel) package which contains modified versions of
+[*multicore*](https://cran.r-project.org/package=multicore) and [*snow*](https://cran.r-project.org/package=snow) .
+
+
+A basic objective of [*BiocParallel*](http://bioconductor.org/packages/BiocParallel) is to reduce the complexity faced when developing
```

Comment:
Sometimes the *BiocParallel* link is to `http://` and sometimes to `https://`. Also, this link is repeated a number of times in the vignette. In these cases it might make sense to the alternative form of adding hyperlinks, where in the body of the text you write
```
A basic objective of [*BiocParallel*][] is to reduce the 
```
and somewhere else in the text (e.g., at the very end) you add
```
[*BiocParallel*]: https://bioconductor.org/packages/BiocParallel
```
This is form of hyperlink is both easier to read in the text, and ensures that the link is always consistent.